dist/aurelia-templating.d.ts

```diff
@@ -472,10 +466,29 @@ export declare class ViewEngineHooksResource {
 }
 export declare function viewEngineHooks(target?: any): any;
 
-/**
- * Dispatches subscribets to and publishes events in the DOM.
- * @param element
- */
+
+export declare interface EventHandler {
+  eventName: string;
+  bubbles: boolean;
+  dispose: Function;
+  handler: Function;
+}
+
+export declare interface SubscriptionHandlerConfig {
+  handler: Function
+  capture?: boolean
+  passive?: boolean
```

Comment:
Whats this passive prop for? I see it nowhere handled in code

Reply:
It is used when we pass the whole object to `EventTarget.prototype.addEventListener` third paramter. Same with `passive` in spec.